jerry-core/jrt/jrt.h

```diff
@@ -60,8 +59,7 @@
 #define JERRY_STATIC_ASSERT_GLUE_(a, b, c) a ## b ## c
```

Comment:
Wouldn't it be better to add an extra underscore (_) ?

    #define JERRY_STATIC_ASSERT_GLUE_(a, b, c) a ## b ## _ ## c